shared/gh/partials/tenant-admin/admin-delete-series-modal.html

```diff
@@ -33,7 +33,7 @@ <h4 class="modal-title" id="modal-title">Are you sure?</h4>
             </div>
             <div class="modal-footer">
                 <button type="button" class="btn btn-link pull-left" data-dismiss="modal">Cancel</button>
-                <button type="button" id="gh-delete-series-delete" class="btn btn-default pull-right">Delete</button>
+                <button type="button" id="gh-delete-series-delete" class="btn btn-default pull-right">Remove</button>
```

Comment:
This should be context sensitive. When a borrowed series is being removed the label should be `Remove`. Otherwise, the label should read `Delete`